pytheus/metrics.py

```diff
@@ -48,15 +53,21 @@ def _validate_labels(self, labels: Sequence[str]):
 
 # class or function based creation ?
 class Metric:
-
     def __init__(
         self,
         collector: MetricCollector,
         labels: Labels | None = None,
+        backend_class: Optional[Backend] = None,
+        backend_config: Optional[BackendConfig] = None,
     ) -> None:
         self._collector = collector
         self._labels = labels
         self._can_observe = self._check_can_observe()
+        if backend_class is not None and backend_config is not None:
+            self._backend = backend_class(backend_config)
```

Comment:
`metric_value_backend` could be a better fit for the backend name

Reply:
0e9845a85d04f65879c8e5a12d369a98d0207c42

Reply:
0e9845a85d04f65879c8e5a12d369a98d0207c42